### PR TITLE
Title: Create an issue template for Issue #389

### PR DESCRIPTION
**Pull request by JamieBort**
Why: To improve documentation.

Creating a  pull request for Issue #389

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## I am submitting a ...
+
+- [ ] bug report
+- [ ] feature request
+- [ ] improvement to documentation
+
+## What is the current behavior?
+
+## If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem
```
Comment (lucavallin) [inline .github/ISSUE_TEMPLATE/issue_template.md:9]:
```suggestion
## If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.
```

**Comment by JamieBort** [comment]
I am new to collaborating. I am not sure if I there is more for me to do, such as update the branch (see screenshot).

Screen Shot 2024-04-02 at 15 55 14

Or if there is nothing more you need me to do. Thanks.

**Review by lucavallin — APPROVED** [review]
Nice, thanks for adding this!

**Comment by JamieBort** [comment]
I'm glad to help.

**Pull request merged**